Assets/Mirror/Runtime/Transport/Transport.cs

```diff
@@ -36,24 +28,27 @@ public abstract class Transport : MonoBehaviour
         #region Client
         /// <summary>
         /// Notify subscribers when when this client establish a successful connection to the server
+        /// <para>callback()</para>
         /// </summary>
-        [HideInInspector] public UnityEvent OnClientConnected = new UnityEvent();
+        public Action ClientConnectedCallback;
```

Comment:
please don't rename them :)
OnClientConnected is short and concise and what we always used.
Renaming can be discussed in a separate PR if at all.
Let's just get this merged without the name change first please